packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -145,7 +145,7 @@ export const Dropdown = memo(
       (group: DropdownItemGroup) => {
         return (
           <>
-            <ListGroupHeader>{group.label}</ListGroupHeader>
+            <ListGroupHeader>{group.label.toUpperCase()}</ListGroupHeader>
```

Comment:
🍹 Maybe instead of `.toUpperCase` we might just want to add `text-transform: uppercase;` instead?

Reply:
The element is not recognizing this style for some reason. Troubleshooting now.